### ISSUE TITLE
Buttons use title attribute for accessible names

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#713]
**Describe the recommendation**

This is NOT a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value), but rather a recommendation.

Interactive controls need accessible names. When there's a properly implemented visible text label it'll serve as an accessible name as well, and no further action is needed.

There are several buttons in the chat widget which only use icons as visible labels, and which then use the `title` attribute to provide a text label. For example the "Send the message" button pictured below:

![Send message button lacking accessible name in devtools, screenshot.](https://github.com/user-attachments/assets/5929b78d-a3ef-44fb-a900-03b1a15ab82e)

The `title` attribute is intended to provide advisory information in addition to the regular label. Although a lot of assistive technology will parse the attribute as an accessible name and announce it to users, this isn't its intended purpose and it shouldn't be relied upon for it.

**To Reproduce**

1. Target the send message button in devtools

**Recommended solution**

Use [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label) instead of `title` to provide accessible names when no visible text label is used.

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: TBA

**Desktop:**

 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This recommendation was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
